src/main/java/com/google/devtools/build/lib/util/OsUtils.java

```diff
@@ -26,7 +26,11 @@ private OsUtils() {
   }
 
   public static String executableExtension(OS os) {
-    return os == OS.WINDOWS ? ".exe" : "";
+    return executableExtension(os == OS.WINDOWS);
+  }
+
+  public static String executableExtension(boolean isWindows) {
+    return isWindows ? ".exe" : "";
   }
 
   /**
```

Comment:
Please mark the following function `executableExtension()` @Deprecated. There are some uses left, that should be cleaned up if we want C++ platformization to fully work.